src/clj/game/cards-operations.clj

```diff
@@ -552,6 +552,18 @@
     :msg "do 1 net damage"
     :effect (effect (damage eid :net 1 {:card card}))}
 
+   "Observe and Destroy"
+   {:req (req (and (pos? (:tag runner))
```

Comment:
Does `:tag` always have a value?  `pos?` will throw a NPE if `(:tag runner)` is `nil`. Maybe best to make this `(:tag runner 0)`